tests/lib/rules/object-property-newline.js

```diff
@@ -506,6 +570,111 @@ ruleTester.run("object-property-newline", rule, {
+            errors: [
+                {
+                    message: "Object properties must go on a new line. The opening bracket of a computed property name may end a line on which another property appears.",
+                    type: "ObjectExpression",
+                    line: 4,
+                    column: 4
+                }
+            ]
+        },
+
+        // noCommaFirst: true
+        {
+            code: "var obj = {\nk1: 'val1'\n, k2: 'val2'\n, k3: 'val3'\n};",
+            output: "var obj = {\nk1: 'val1'\n,\nk2: 'val2'\n,\nk3: 'val3'\n};",
+            options: [{ noCommaFirst: true }],
+            errors: [
+                {
+                    message: "Object properties must go on a new line. The comma delimiting two properties may not share a line with any of the second property.",
+                    type: "ObjectExpression",
+                    line: 3,
+                    column: 3
+                },
+                {
+                    message: "Object properties must go on a new line. The comma delimiting two properties may not share a line with any of the second property.",
+                    type: "ObjectExpression",
+                    line: 4,
+                    column: 3
+                }
+            ]
+        },
+        {
+            code: "var obj = {\nk1: 'val1'\n, k2: 'val2'\n, [\nbaz1\n]: 'val3'\n};",
+            output: "var obj = {\nk1: 'val1'\n,\nk2: 'val2'\n,\n[\nbaz1\n]: 'val3'\n};",
+            options: [{ noCommaFirst: true }],
+            parserOptions: { ecmaVersion: 6 },
+            errors: [
+                {
+                    message: "Object properties must go on a new line. The comma delimiting two properties may not share a line with any of the second property.",
+                    type: "ObjectExpression",
+                    line: 3,
+                    column: 3
+                },
+                {
+                    message: "Object properties must go on a new line. The comma delimiting two properties may not share a line with any of the second property.",
+                    type: "ObjectExpression",
+                    line: 4,
+                    column: 3
+                }
+            ]
+        },
+
+        // { treatComputedPropertiesLikeJSCS: true, noCommaFirst: true }
+        {
+            code: "var obj = {\nk1: 'val1'\n, k2: 'val2'\n, [\nbaz2\n]: 'val3'\n};",
+            output: "var obj = {\nk1: 'val1'\n,\nk2: 'val2'\n, [\nbaz2\n]: 'val3'\n};",
+            options: [{
+                treatComputedPropertiesLikeJSCS: true, noCommaFirst: true
+            }],
+            parserOptions: { ecmaVersion: 6 },
+            errors: [
+                {
+                    message: "Object properties must go on a new line. The opening bracket of a computed property name may end a line on which another property appears. The comma delimiting two properties may not share a line with any of the second property.",
```

Comment:
these messages are a bit too long, can be simple?

Reply:
Yes, I can shorten them. I’ll do that.